src/main/java/teammates/logic/backdoor/BackDoorLogic.java

```diff
@@ -152,9 +152,9 @@ public String persistDataBundle(DataBundle dataBundle)
             response = injectRealIds(response);
         }
         frDb.createFeedbackResponses(responses.values());
-        
-        for(FeedbackSessionAttributes session : sessions.values()){
-            updateRespondants(session.feedbackSessionName, session.courseId);
+
+        for (FeedbackResponseAttributes response : responses.values()) {
+            updateRespondants(response.feedbackSessionName, response.courseId);
```

Comment:
Remove the modifications to this file, if you can, since in the end nothing was changed here (other than indentation). Logically, none of your commits should touch this file.